### DISCUSSION TITLE
How to install on unraid?

### DISCUSSION THREAD

**Discussion by simondsmason** [tphakala/birdnet-go#289] [Q&A]
Hi, I was using Birdcage on unraid but the project is no longer maintained.  I managed to install it using portainer with  lot of help from the community.  I would like to switch to birdnet-go but as a neophyte on this stuff I could use some pointers?  Thanks.

**Comment by tphakala** [comment]
I haven't used unraid nor Birdcage, but as I would expect that BirdNET-Go container would be installed pretty much same way as you did with Birdcage.

**Comment by normal-dot-brad** [comment]
@simondsmason did get it working on Unraid ? If so could you share how you did it. I'm having no luck.

**Reply by simondsmason** [reply]
I haven't given it much attention yet.  I had a challenging time getting the original installed - going back and forth with a Good Samaritan on GitHub to get it working.  Hopefully there is someone here who could offer step by step in Unraid?

**Reply by normal-dot-brad** [reply]
Ok thanks, if I figure it out I'll post here

**Comment by AKHwyJunkie** [comment]
An easy way to make this work on Unraid would be to simply create a VM with Debian or Ubuntu as the OS.  Then, use the "Quick Install Script" [found here](https://github.com/tphakala/birdnet-go).  (You could also do a manual docker install, or a fully compiled build too.)  I did a similar thing with my Proxmox based install, although was able to use a container in this case.  I went through the entire install.sh script prior to installing it and it's totally benign.

**Comment by TimNiceButDim** [comment]
Really wish this was available on Unraid in the community apps. I would love to install it using docker compose but I know I will probably break something.

Really hope it can be added one day, as I would love to use this amazing app!

**Comment by uSlackr** [comment]
Talk to the unraid app people.

\\Greg
________________________________
